assignments/objects.js

```diff
@@ -4,11 +4,47 @@
 // HR needs some information on the new interns put into a database.  Given an id, email, first name, and gender. Create an object for each person in the company list:
 
 // 1,mmelloy0@psu.edu,Mitzi,F
+const Mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
 // 2,kdiben1@tinypic.com,Kennan,M
+const Kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": () => {
+    return 'Hello, My name is ' +  Kennan.name;
+  },
+}
+
 // 3,kmummery2@wikimedia.org,Keven,M
+const Keven = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
 // 4,gmartinson3@illinois.edu,Gannie,M
+const Ganniee = {
+  "id": 4,
+  "name": "Ganniee",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
 // 5,adaine5@samsung.com,Antonietta,F
-
+const Antonietta = {
+  "id": 5,
+  "name": ",Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  "multiplyNums" : (num1, num2) => {
+    return num1 * num2; 
+  }
```

Comment:
good job, but remember with the ES6 syntax fat arrow functions you have a access to an implicit return: 
```suggestion
  "multiplyNums" : (num1, num2) => num1 * num2;
```
does the same thing without having to be explciit and saves you a couple lines